### ISSUE TITLE
New accessible document format

### ISSUE THREAD

**Issue by NellySSJ** [NHSDigital/DataDictionaryPublication#594]
WCAG requirements mention PDF documents are not accessible, and there should be an open document format (e.g.HTML) available for users with accessibility needs.
We need to understand how this impacts to CR PDF documents that are created, and discuss potential ways forward.

Things to consider...

- Generating new accessible format output:
Converting PDF to HTML using Adobe Pro - https://www.adobe.com/acrobat/hub/how-to-convert-pdf-to-webpage.html
DDPM generating HTML file using DITA

- Where to store the new accessible file type

- Availability of new accessible file type
DDCN on Data Dictionary Release HIstory page - https://www.datadictionary.nhs.uk/whats_new.html
Information Standards and Data Collections pages - https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections

**Comment by AngelaFaulding** [comment]
Another page affected is: https://www.[datadictionary.nhs.uk/DDCNs/index.html

**Comment by NellySSJ** [comment]
Please move to backlog for now.
